cfn-resources/autogen/schema-gen/template.go

```diff
@@ -43,7 +45,7 @@ type RequireParam struct {
 }
 
 type Definitions struct {
-	Type                 string              `json:"type,omitempty"`
+	Type                 *openapi3.Types     `json:"type,omitempty"`
```

Comment:
Looks like a [change in 0.124.0](https://github.com/getkin/kin-openapi?tab=readme-ov-file#v01240)

Uncertain if we use this at all?

Reply:
Seems like it only affect the generation not the resources themselves.

Reply:
>Looks like a [change in 0.124.0](https://github.com/getkin/kin-openapi?tab=readme-ov-file#v01240)

right, this upgrade is from 0.123 to 0.125

>Uncertain if we use this at all?
it is used when creating properties & definitions